lib/wor/paginate/paginate.rb

```diff
@@ -0,0 +1,39 @@
+module Wor
+  module Paginate
+    ADAPTERS = [Adapters::KaminariAlreadyPaginated,
+                Adapters::WillPaginateAlreadyPaginated,
+                Adapters::WillPaginate,
+                Adapters::Kaminari,
+                Adapters::ActiveRecord,
+                Adapters::Enumerable].freeze
+
+    def render_paginated(content, options = {})
+      render json: paginate(content, options)
+    end
+
+    def paginate(content, options = {})
+      adapter = find_adapter_for_content(content, options)
+      raise Exceptions::NoPaginationAdapter unless adapter.present?
+      formatter_class(options).new(adapter, options).format
+    end
+
+    def formatter_class(options)
+      options[:formatter].presence || Formatter
+    end
+
+    def find_adapter_for_content(content, options)
+      ADAPTERS.map do |possible_adapter|
```

Comment:
I think it is more readable this way

    ADAPTERS.map { |adapter| adapter.new(content, page(options), limit(options)) }
            .find(&:adapt?)